config/settings.yml

```diff
@@ -11,3 +11,8 @@ workflow:
   url: 'http://localhost:3001'
 
 staging_location: '/dor/workspace'
+
+
+# This user is allowed to proxy other users, we expect that it'll be argo and it
+# will be responible for only proxying users which it has verififed with Shibboleth.
+argo_user: argo@dlss.sul.stanford.edu
```

Comment:
this email should be clearly a dummy email (e.g. `overwritten@example.org`) or it should be a real one.  I would lean towards a real one, maybe sent to team list so if it gets used we become aware things aren't working right.

Reply:
@ndushay I need it to uniquely identify argo (and not other systems).  Which one would you suggest?

Reply:
@andrewjbtw is there an existing argo email that makes sense?  should we have one for stage also?   Or just use dummy email addresses?

Reply:
- will the email above EVER get any email?  if no, could be something like `proxying-user@argo-prod.stanford.edu`. If yes ... who should get the email?  should it be list you set up with mailman that's for devs + Andrew?

- do we need it to be different for prod/stage/local development?  If so, the comment should indicate and/or it should be in shared_configs or something.  If no, then `proxying-user-4-argo@dlss.sul.stanford.edu` or some such would be good.

Reply:
no email is being sent here, it's just used to identify a service or a user.  It doesn't need to change in different environments.

Reply:
then cool.  do as you wish.